cmake/MiscCheck.cmake

```diff
@@ -17,20 +17,22 @@ set(CAFFE2_USE_EXCEPTION_PTR 1)
 # is being deprecated. As a result, we will test if this is the environment we
 # are building under. If yes, we will turn off deprecation warning for a
 # cleaner build output.
```

Comment:
Can we just delete the whole block? The comment mentions ubuntu 14.04 which is no longer relevant.